apps/web/app/(app)/onboarding/OnboardingNextButton.tsx

```
@@ -0,0 +1,22 @@
+"use client";
+
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import { useSearchParams } from "next/navigation";
+
+export function OnboardingNextButton() {
+  const searchParams = useSearchParams();
+  const stepParam = searchParams.get("step");
+  const currentStep = stepParam ? Number.parseInt(stepParam) : 1;
+  const nextStep = isNaN(currentStep) ? 2 : currentStep + 1;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)